### PR TITLE
Add Broadcast.Show check to CustomItems

### PR DESCRIPTION
**Pull request by FUTURE-SL**

### CODE REVIEW

**Review by louis1706 — APPROVED** [review]

**Review by IRacle1 — APPROVED** [review]

File: Exiled.CustomItems/API/Features/CustomItem.cs [hunk 1]
```diff
@@ -978,7 +978,8 @@ protected virtual void OnWaitingForPlayers()
         /// <param name="player">The <see cref="Player"/> who will be shown the message.</param>
         protected virtual void ShowPickedUpMessage(Player player)
         {
-            player.ShowHint(string.Format(Instance!.Config.PickedUpHint.Content, Name, Description), Instance.Config.PickedUpHint.Duration);
+            if (Instance!.Config.PickedUpHint.Show)
```
Comment (Vladislav-CS) [inline Exiled.CustomItems/API/Features/CustomItem.cs:981]: Null check is not needed for Instance, but for hint

File: Exiled.CustomItems/API/Features/CustomItem.cs [hunk 2]
```diff
@@ -987,7 +988,8 @@ protected virtual void ShowPickedUpMessage(Player player)
         /// <param name="player">The <see cref="Player"/> who will be shown the message.</param>
         protected virtual void ShowSelectedMessage(Player player)
         {
-            player.ShowHint(string.Format(Instance!.Config.SelectedHint.Content, Name, Description), Instance.Config.SelectedHint.Duration);
+            if (Instance!.Config.SelectedHint.Show)
```
Comment (Vladislav-CS) [inline Exiled.CustomItems/API/Features/CustomItem.cs:991]: Same

**Review by Vladislav-CS — CHANGES_REQUESTED** [review]

File: Exiled.CustomItems/API/Features/CustomItem.cs (hunk 2 above)
Comment (FUTURE-SL) [reply Exiled.CustomItems/API/Features/CustomItem.cs:991]: image
😎

File: Exiled.CustomItems/API/Features/CustomItem.cs (hunk 1 above)
Comment (IRacle1) [reply Exiled.CustomItems/API/Features/CustomItem.cs:981]: no, everything is alright
hint cant be null
plugin is nullable

**Comment by FUTURE-SL** [comment]
"Random optimization" added to new pull request, sry by revert

**Review by louis1706 — APPROVED** [review]

**Review by Thundermaker300 — APPROVED** [review]

**Pull request merged**